conans/test/functional/command/profile_test.py

```diff
@@ -75,8 +70,9 @@ def profile_update_and_get_test(self):
         client = TestClient()
         client.run("profile new ./MyProfile --detect")
         if "WARNING: GCC OLD ABI COMPATIBILITY" in client.out:
-            self.assertIn("edit the MyProfile profile at", client.out)
-            self.assertIn(os.path.join(client.current_folder, "MyProfile"), client.out)
+            self.assertIn("Or edit '{}/MyProfile' and "
```

Comment:
The output changed some time ago to contain the abs path, but the test was never failing in CI because of the bug with the major version of the GCC, the `if "WARNING: GCC OLD ABI COMPATIBILITY" in client.out:` was never true.